codechecker_lib/arg_handler.py

```diff
@@ -334,6 +336,70 @@ def handle_quickcheck(args):
         shutil.rmtree(args.workspace)
 
 
+def consume_plist(item):
+    plist, args, context = item
+    LOG.info('Consuming ' + plist)
+
+    action = build_action.BuildAction()
+    action.analyzer_type = analyzer_types.CLANG_SA
+    action.original_command = 'Imported from PList directly'
+
+    rh = analyzer_types.construct_result_handler(args,
+                                                 action,
+                                                 context.run_id,
+                                                 args.directory,
+                                                 {},
+                                                 None,
+                                                 not args.stdout)
+    
+    rh.handle_plist(os.path.join(args.directory, plist))
+
+def handle_plist(args):
+
+    context = generic_package_context.get_context()
+    context.codechecker_workspace = args.workspace
+    context.db_username = args.dbusername
+
+    if not args.stdout:
+        args.workspace = os.path.realpath(args.workspace)
+        if not os.path.isdir(args.workspace):
+            os.mkdir(args.workspace)
+    
+        check_env = analyzer_env.get_check_env(context.path_env_extra,
+                                               context.ld_lib_path_extra)
+    
+        sql_server = SQLServer.from_cmdline_args(args,
+                                                 context.codechecker_workspace,
+                                                 context.migration_root,
+                                                 check_env)
+    
+        conn_mgr = client.ConnectionManager(sql_server,
+                                            'localhost',
```

Comment:
localhost is hard coded because the report storage server is started always where the analysis is done on the same host.
Currently it does not support the use case where the storage server is started on a separate machine. It reduces configuration complexity because the storage server is started automatically.